### PR TITLE
Why Be Concerned About AI Safety?

### PR DESCRIPTION
**Pull request by botahamec**
#### Preliminary Checks - Formatting and Structure 
- [ ] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [ ] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### CODE REVIEW

**Comment by botahamec** [comment]
I actually proposed this idea before we proposed topics through issues, so there isn't an issue to link :sweat_smile:

File: articles/ai-safety/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+* **Make an AI and tell it to maximize [QALYs](https://en.wikipedia.org/wiki/Quality-adjusted_life_year).** Utilitarianism is pretty popular, but it still has [problems](https://www.thinkingmuchbetter.com/main/5-objections-to-utilitarianism). It also still has the problem of [population ethics](https://www.youtube.com/watch?v=vqBl50TREHU). Also, as mentioned before, AIs can take advantage of their reward function. If this AI was implemented, all it would do is maximize the number of babies born.
+
+### Organizations to Donate To
+
+[In](https://80000hours.org/) [Effective](https://www.evidenceaction.org/) [Altruism](https://www.gfi.org/), [we](https://d-rev.org/) [like](https://www.givedirectly.org/) [to](https://www.gofundme.com/f/covidmoonshot) [focus](https://www.ciwf.com/) [on](https://aisafetycamp.com/) [organizations](https://www.againstmalaria.com/) [to](https://allfed.info/) [donate](https://www.charitywater.org/) [to](https://www.arborday.org/). Here's a shortlist of places to look at:
```
Comment (lalith1403) [inline articles/ai-safety/index.md:46]: Its not clear why you have inserted links in the first sentence. Each word has a different link. Do you want to emphasize on that? I don't quite understand that. If they are links to organizations, then probably add them in the list below. If the list is too big, filter it out accordingly.

**Review by lalith1403 — APPROVED** [review]
Just a minor update and this is good to go. :)

**Comment by botahamec** [comment]
@lalith1403 Sorry it's been a while. Should be good now.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**